app/javascript/Pages/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,171 @@
+.navbar {
+  background: #6600cc;
+  height: 80px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.2rem;
+  position: sticky;
+  top: 0;
+  z-index: 999;
+}
+
+.navbar-container {
+  display: flex;
+  justify-content: space-between;
+  height: 80px;
+}
+
+.container {
```

Comment:
**suggestion (blocking)**: we prefer to write styles that are shared across the whole application as it gives you much more reusability. In that sense you need to be careful with naming classes the same as in other files. That's why we like BEM formatting, because all the classes related with navbar should start with `navbar`